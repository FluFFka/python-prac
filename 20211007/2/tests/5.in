(18, 51, -15, 43, 67, 66, -46, -32, -49, 33, -78, 81, -21, -93, 16, 1, 17, 92, -66, -28, 51, -100, -24, -6, 32, 63, 57, -25, 31, 19, -56, 79, 42, 6, -96, 85, 97, -91, -97, -72, 67, -99, 60, 77, 64, -72, -33, -12, 47, -61, 1, -15, -3, 66, -1, -98, -21, -79, -3, -93, 64, -99, 71, 54, 40, 35, 0, 39, 46, 22, 60, -31, 77, -29, 4, 53, 12, -35, -18, -99, 91, 15, 85, -21, -95, -57, -5, 47, 59, -77, -78, -9, -73, 13, 61, -17, -3, -98, -3, -33, 0, -17, -97, -81, 72, 6, -70, 29, 24, -11, -54, 83, -41, -25, 21, 48, 97, -54, 100, 96, -91, -55, -65, -98, -42, -30, 32, -72, 15, 87, -12, -82, -42, -6, -73, -79, -6, -56, 93, 78, 19, -78, 58, 47, 25, -51, -39, 10, -13, -99, -32, -55, 39, -76, 65, 82, 94, 16, 1, 48, 21, 46, -1, -34, 75, 31, 92, -67, 22, 75, 92, 45, 53, 85, -72, 98, -74, 13, 54, -85, -46, -8, -6, 13, -13, 75, -82, 8, -57, 93, -64, 35, 73, -54, 5, 98, 86, -12, 82, 11), (-108, -1, -40, -95, -31, -3, -87, 120, 38, 93, -28, -18, -75, -27, 22, 92, -67, -24, -43, 63, 94, -103, -19, -25, -87, -50, 68, 110, -63, 72, -13, 8, -6, 58, 29, 63, -30, 45, -45, -96, -25, -15, 67, 63, 60, 62, 25, 30, 113, 99, -101, -26, -45, 22, 87, -37, -2, -93, 103, 110, 110, 32, -94, -39, -48, -84, 89, -67, 7, -50, -76, 67, 88, 16, -71, -11, -61, 68, 22, 0, -40, 98, -80, 69, -17, -100, -90, 119, -25, 32, -105, -105, 2, 81, 3, 112, 54, -86, 116, 78)
