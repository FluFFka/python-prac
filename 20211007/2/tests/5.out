(18, 51, 43, 66, -46, -32, -49, 33, -78, -21, 1, 17, -66, 51, 57, 31, 19, -56, 79, 42, 6, 85, 97, -91, -97, -72, -99, 77, 64, -72, -33, -12, 47, 1, 66, -98, -21, -79, 64, -99, 71, 40, 35, 39, 46, 77, -29, 4, 53, 12, -35, -99, 91, 15, 85, -21, -57, -5, 47, 59, -77, -78, -9, -73, 13, 61, -98, -33, -97, -81, 6, -70, 24, -54, 83, -41, 21, 48, 97, -54, 100, 96, -91, -55, -65, -98, -42, -72, 15, -12, -82, -42, -73, -79, -56, 19, -78, 47, -51, 10, -99, -32, -55, 39, 65, 82, 1, 48, 21, 46, -34, 75, 31, 75, 53, 85, -72, -74, 13, -85, -46, -8, 13, 75, -82, -57, -64, 35, 73, -54, 5, 86, -12, 82, 11)
